content/en/real_user_monitoring/guide/shadow-dom.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Enrich Your Session Replays With Shadow DOM Components
+kind: guide
+description: Guide about Shadow DOM compatibility with Session Replay.
+further_reading:
+- link: '/real_user_monitoring/session_replay/'
+  tag: 'Documentation'
+  text: 'Learn about Session Replay'
+---
+
+## Overview
+
+Shadow DOM helps developers build more modern websites by allowing them to incorporate isolated and reusable components into their code. Often used in order to keep a clean code structure and avoid style clashes, Shadow DOM usage has become more prominent in modern web development practices. 
+
+Starting with `v4.31.0` of the [RUM Browser SDK][1], Datadog provides support of open Shadow DOM without any extra configuration required — components which are located inside a shadow root are automatically captured by Session Replay.
+
+**Note**: The open Shadow DOM compatibility has been tested on popular frameworks.
+
+## What is not supported
```

Comment:
Similar to my feedback on the other PR, is it possible for us to instead list the things we do support?

Reply:
@victorien-provenzano Are we planning on supporting the other modes? Maybe we could add an alert box at the top to highlight that only the open mode is supported. Or if we are not planning on supporting other modes, change the title to `Open Shadow DOM`

Reply:
@estherk15 thanks for the suggestion, went ahead and added an alert box!